cargo-near/src/commands/new/mod.rs

```diff
@@ -100,6 +102,10 @@ impl NewContext {
             ));
         }
 
+        let _detached_thread_handle = std::thread::Builder::new().spawn(|| {
+          util::track_usage()
+        }).unwrap(); //.join();
```

Comment:
@frol @dj8yfo 
without joining here it seems like the `track_usage` just isn't invoked